src/screens/Home/ActionButtons.js

```diff
@@ -159,19 +167,33 @@ class ActionButtons extends React.Component<Props, State> {
 
     switch (visibleActionModal) {
       case RECEIVE:
-        return accountsInfo.map(({
-          type,
-          formattedBalance,
-          additionalInfo,
-          address,
-        }) => ({
-          key: type,
-          value: formattedBalance,
-          ...additionalInfo,
-          onPress: () => this.setState({ receiveAddress: address }),
-          label: `To ${additionalInfo.title}`,
-        }),
-        );
+        return [
+          {
+            key: 'buy',
+            label: 'Buy with card or Apple Pay',
```

Comment:
Hm, maybe we should show "Apple Pay" to iOS users only? :)

Reply:
well the Or clause is probably grammatically separating the possibilities, but yea might be an option if product team wants that (anyway will these strings be extracted for transaction ?)

Reply:
Yeah, they will be, eventually :).
Still, this will be going out before we merge in all translations. 
And on Android, in my opinion, it seems a bit weird to see 'Apple Pay' option :)